Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Prisma_Cloud_Enrichment.yml

```
@@ -383,8 +355,208 @@ tasks:
     view: |-
       {
         "position": {
-          "x": 890,
-          "y": 1040
+          "x": 630,
+          "y": 990
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "10":
+    id: "10"
+    taskid: cf6daeee-77c3-495d-870e-e0d15087e37a
+    type: regular
+    task:
+      id: cf6daeee-77c3-495d-870e-e0d15087e37a
+      version: -1
+      name: Set flag for completed enrichment
```

```suggestion
      name: Set true flag for completed enrichment
```
Consistency with other playbooks